### PR TITLE
Bump v0.44.2

### PR DESCRIPTION
**Pull request by ali-ramadhan**
Been a lot of changes since v0.44.1 (including some bug fixes) but don't think there have been any breaking changes (or massive new features) so was thinking we should tag and release v0.44.2.

### CODE REVIEW

**Review by navidcy — APPROVED** [review]
agree; doesn't seem to be any bug fixes or breaking changes here:
https://github.com/CliMA/Oceananigans.jl/compare/v0.44.1...ali/bump-v0.44.2

**Comment by ali-ramadhan** [comment]
I'll wait until #1198 and #1199 are merged then tag and release.

**Comment by ali-ramadhan** [comment]
Status of #1198 is unclear so I'll merge to tag and release since v0.44.2 includes at least a few bug fixes.

**Pull request merged**
